app/src/main/java/com/example/taskscheduler/data/repos/NoteRepositoryImpl.kt

```diff
@@ -48,7 +54,11 @@ class NoteRepositoryImpl(
         }
     }
 
-    override suspend fun fetchNotes(notesListItem: NotesListItem, listNotes: List<Note>, scope: CoroutineScope) = suspendCancellableCoroutine { continuation ->
+    override suspend fun fetchNotes(
+        notesListItem: NotesListItem,
+        listNotes: List<Note>,
+        scope: CoroutineScope
+    ) = suspendCancellableCoroutine { continuation ->
```

Comment:
```suggestion
    ) = suspendCancellableCoroutine { _ ->
```
Похоже continuation не юзается